scenarioo-client/app/app.module.ts

```diff
@@ -123,7 +123,7 @@ import {BranchAliasesComponent} from './manage/branch-aliases/branch-aliases.com
         DiffInfoService,
         SharePageURL,
         Location,
-        FilterPipe,
+        FilterArrayPipe,
```

Comment:
Why do you need to import your pipe as a service?

Reply:
I need to inject the pipe into a class, therefore I provide it in the providers array of ngModule. I read that in the angular documentation (https://angular.io/guide/pipes#pipes). Do you see a better way?